components/toast/index.css

```diff
@@ -10,18 +10,60 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+/* toast/index.css
+ * 
+ * .spectrum-Toast::after is the Focus ring
+ *
+ * Component naming template:
+ * `--system-component-style-description-modifier`
+ *
+ * Example:
+ * `--spectrum-toast-spacing-text-to-button-horizontal`
+ */
+
 .spectrum-Toast {
-  --spectrum-toast-icon-padding-y: var(--spectrum-global-dimension-size-85);
-  --spectrum-toast-neutral-content-padding-top: var(
-    --spectrum-global-dimension-size-65
-  );
-  --spectrum-toast-content-padding-bottom: var(
-    --spectrum-global-dimension-size-65
-  );
-
-  --spectrum-toast-button-margin-right: var(
-    --spectrum-global-dimension-size-130
-  );
+  /* Hardcoded variables */
+  --spectrum-toast-font-weight: 400;
```

Comment:
maybe you could use this custom-var for now:
https://github.com/adobe/spectrum-css/blob/main/components/tokens/custom-spectrum/custom-vars.css#L32
`  --spectrum-font-weight-regular: 400;`